### DISCUSSION TITLE
installing MoBI 7.3

### DISCUSSION THREAD

**Discussion by praveenniper** [Open-Systems-Pharmacology/Forum#782] [Questions & Problems]
Hi All, when I install the full  OSPSuite-Full.9.1.3.exe in my windows computer, it seems only PK-Sim is getting installed in the program folder. I tried uninstalling and reinstalling a few times but could not install MobI7.3. 

I could install only the following item: 
-MikeTex2.9.3

Thank you in advance for your help and time in advance. 

Best wishes,

Praveen

**Comment by msevestre** [comment]
Hi @praveenniper 

Why are you trying to start MoBi 7.3? The version installed with the OSPSuite 9.1 will be Mobi 9.1
Can you simply search for MoBi in your startup command ?(press the windows key on your keyboard and enter MoBi)

Alternatively, create a model in PKSim and export it to mobi (right click on the simulation send to mobi). Do you have any error message?

**Comment by praveenniper** [comment]
Hi @msevestre thank you, it seems to be working now :) 

Thank you, 

Praveen
